install.sh

```diff
@@ -44,8 +44,7 @@ if [[ ${STANDALONE,,} == "yes" ]]; then
     echo "LTSP server will be in standalone mode of operation"	
     echo "LTSP server will provide DHCP services.."	
     sed -i /etc/dnsmasq.d/ltsp-server-dnsmasq.conf \
-	-e "/192.168.1.0,proxy\$/ c #dhcp-range=${NETWORK}.0,proxy" \
-	-e "/10.0.2.0,proxy\$/ c #dhcp-range=10.0.2.0,proxy"
+	-e "/.*,proxy\$/ c #dhcp-range=" 
```

Comment:
This is not the right way of doing it. If you do it like this why don't you delete it at all? What is the purpose of making it a useless comment.

Reply:
Yes you are right.